metadata.json

```diff
@@ -115,7 +115,7 @@
     }
   ],
   "dependencies": [
-    {"name":"puppetlabs-stdlib","version_requirement":">= 5.1.0 < 6.0.0"},
+    {"name":"puppetlabs-stdlib","version_requirement":">= 5.1.0 < 7.0.0"},
     {"name":"puppetlabs-inifile","version_requirement":">= 2.4.0 <= 3.0.0"},
     {"name":"puppetlabs-apt","version_requirement":">= 6.0.0 < 7.0.0"}
```

Comment:
puppetlabs-apt should be < 8.0.0 because apt (<7.0.1) has as dependency stdlib < 6.0.0